src/TestFramework/Microsoft.Rest.ClientRuntime.Azure.TestFramework/TestEnvironmentFactory.cs

```diff
@@ -26,307 +27,6 @@ public static class TestEnvironmentFactory
         /// Sample Value 2 - Prompt for login credentials:
         /// TEST_CSM_ORGID_AUTHENTICATION=SubscriptionId={subscription-id};AADAuthEndpoint=https://login.windows-ppe.net/;BaseUri=https://api-next.resources.windows-int.net/
         /// </summary>
-        const string TestCSMOrgIdConnectionStringKey = "TEST_CSM_ORGID_AUTHENTICATION";
```

Comment:
As discussed, we are keeping this.

---
In reply to: [78658036](https://github.com/Azure/azure-sdk-for-net/pull/2328#discussion_r78658036) [](ancestors = 78658036)